lib/core/address_validator.dart

```diff
@@ -301,6 +307,8 @@ class AddressValidator extends TextValidator {
         return '([^0-9a-zA-Z]|^)[1-9A-HJ-NP-Za-km-z]{43,44}([^0-9a-zA-Z]|\$)';
       case CryptoCurrency.trx:
         return '(T|t)[1-9A-HJ-NP-Za-km-z]{33}';
+      case CryptoCurrency.zano:
+        return '[0-9a-zA-Z]{1,100}';
```

Comment:
it might look like this:

([1-9A-HJ-NP-Za-km-z]{90,200})|(@[\w\d\-\.]+)

this includes regular zano addresses like this: 
ZxBvJDuQjMG9R2j4WnYUhBYNrwZPwuyXrC7FHdVmWqaESgowDvgfWtiXeNGu8Px9B24pkmjsA39fzSSiEQG1ekB225ZnrMTBp
or alias like this: 
@ravaga

Reply:
updated